portfolio/src/main/webapp/script.js

```diff
@@ -39,21 +39,20 @@ function randQuote() {
 // Fetch comments from DataServlet.java
 function getComments() {
 
-  fetch('/comments').then(response => response.json()).then((comments) => {
+  fetch('/comments').then(response => response.json()).then((cmtContainer) => {
 
-    var str_comments = "<h2>Comments</h2><p>";
+    if(!cmtContainer) return;
 
-    for (var index in comments){
-      str_comments += comments[index] + "<br>";
+    var str_comments = "<h2>Comments</h2>";
```

Comment:
Prefer creating DOM nodes using the DOM API ([`document.createElement`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) in this case), then appending them to the proper node in the existing tree.

Happy to provide more guidance on this if you'd like it!